site/static/examples/config-with-node-labels.yaml

```diff
@@ -0,0 +1,13 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+- role: worker
+  kubeadmConfigPatches:
+  - |
```

Comment:
should have a comment that this will apply to joined nodes, I'm not sure if the average kind user is aware which those even are though .. :thinking:

Reply:
maybe should also have an example of how to apply a label to the control-plane